### DISCUSSION TITLE
Reset conversation_id

### DISCUSSION THREAD

**Discussion by nukeador** [acheong08/ChatGPT#191] [Q&A] [answered]
Is there a way to reset the conversation ID for an existing Chatbot() instance?

**Comment by acheong08** [comment, marked as answer]
`chatbot.reset_chat()`

**Reply by nukeador** [reply]
Thanks, is it then possible to generate a new Chatbot() with a specific conversation_id?

I'm a bit confused about who handles these conversation_id

I'm trying to set a different id for each conversation and be able to reset them or set them manually to recover a conversation "history".
